modules/ROOT/pages/export-to-exchange-task.adoc

```diff
@@ -3,37 +3,71 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio allows you to connect to Exchange to publish your project as a new asset.
+Anypoint Studio enables you to write documentation in your new projects and to connect to Exchange to publish your project as a new asset.
```

Comment:
```suggestion
Anypoint Studio enables you to write documentation in your new projects and to connect to Exchange to publish your projects as new assets.
```

Reply:
Should this be "write documentation for your new projects" instead of "in your new projects"?

Reply:
Actually, maybe both are right!
You can write documentation _for_ your project _in_ your project. But I like the "for" more :)
Thanks!